recipes/CoVoST/AST/hparams/w2v2_llama3.yaml

```diff
@@ -0,0 +1,256 @@
+# ############################################################################
+# Model: E2E AST with XLS-R and Llama3
+# Encoder: XLS-R
+# Decoder: LLama3
+# Tokens: unigram
+# losses: KLdiv (Label Smoothing loss)
+# Authors:  Titouan Parcollet
+# ############################################################################
+# Seed needs to be set at top of yaml, before objects with parameters are made
+seed: 3407
+__set_seed: !apply:speechbrain.utils.seed_everything [!ref <seed>]
+output_folder: !ref results/conformer_en/<seed>
+output_bleu_folder: !ref <output_folder>/
+save_folder: !ref <output_folder>/save
+train_log: !ref <output_folder>/train_log.txt
+
+# Data files
+data_folder: !PLACEHOLDER  # e.g, /localscratch/cv-corpus-5.1-2020-06-22/fr
+train_tsv_file: !PLACEHOLDER  # Standard CommonVoice .tsv files
+dev_tsv_file: !PLACEHOLDER  # Standard CommonVoice .tsv files
+test_tsv_file: !PLACEHOLDER  # Standard CommonVoice .tsv files
+accented_letters: False
+src_language: en
+tgt_language: de
+train_csv: !ref <output_folder>/train.csv
+valid_csv: !ref <output_folder>/dev_small.csv
+test_csv: !ref <output_folder>/test.csv
+skip_prep: False # Skip data preparation
+convert_to_wav: True # Switch this to True to convert all mp3 files to wav.
+
+# URL for the HuggingFace model we want to load (BASE here)
+wav2vec2_hub: microsoft/wavlm-large
+wav2vec2_folder: !ref <save_folder>/wav2vec2_checkpoint
+wav2vec2_frozen: False
+
+# LLM options
+llm_path: !PLACEHOLDER # /scratch-01/llama-2/llama-2-7b-hf, or /scratch-01/llama-2/llama-2-7b-chat-hf
+llm_prompt: " Translate English speech to German text: "
```

Comment:
Now that I'm thinking about it, but does it make sense to have a prompt if the LoRA adapters are only trained to perform English -> German?